tools/ci-skip-for-maintainers.js

```diff
@@ -0,0 +1,103 @@
+import { readFileSync } from "fs";
```

Comment:
@Udhayakumari The linter is now fixed, thank you for that! On another note though, this file shouldn't  be in here, did you maybe accidentally add it during a rebase? Either way, please remove it from the diff and then we are good to go from my side.